### ISSUE TITLE
Volunteer Frontend 

### ISSUE THREAD

**Issue by Anshul-Birla** [TritonSE/LAP-Student-Tracker#133]
The last task left is the volunteer frontend. Basically, when a volunteer joins up, we want to do something like this:

<img width="661" alt="image" src="https://user-images.githubusercontent.com/42627151/219539291-bb152ee4-a825-4112-9a26-3bd24c523e83.png">

<img width="674" alt="image" src="https://user-images.githubusercontent.com/42627151/219539334-81582434-6e86-4c66-a90a-fdf66d5ef619.png">

<img width="688" alt="image" src="https://user-images.githubusercontent.com/42627151/219539364-dc533cf3-a2b8-4723-8660-60348e95dc0c.png">

You do not have to send any of these fields anywhere - just store them. The app will send them in an email to the people over at LAP. What needs to happen is the various screens need to be set up. 

Basically, we set up the intro screen, the question screens, and the enter availability screen. The enter availability screen should just link to the homepage. There, the volunteer can just add their availability from the manage section. This code should just be the same as the code in the teacher section (you can literally just use the same button and component). There should be absolutely 0 back end work here.
